docs/_articles/en/lwc/linting.md

```diff
@@ -11,16 +11,22 @@ Linting provides errors about malformed code while you edit. VS Code enforces Sa
 
 ## Installation
 
-1. If you created an SFDX project using the `force:project:create` command, the project may contain a `package.json` file with the ESLint plugin already included. Open `package.json` and verify that it contains the following:
+### For a New Project
 
-   ```json
-   "scripts": {
-       "lint": "npm run lint:lwc",
-       "lint:lwc": "eslint force-app/main/default/lwc"
-   }
-   ```
+If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included. Run `npm install` in your project directory to install the dependencies from `package.json`. To start linting, run `npm run lint:lwc`. You must have components in your project in order to run this command.
 
-2. Add the Salesforce ESLint configuration and ESLint to devDependencies.
+### For an Existing Project with a `package.json` File
```

Comment:
An existing project with a `package.json` file could already include the scripts and the devDependencies.

Reply:
@jbleyleSF How about: In `package.json`, check whether `devDependencies` includes the Salesforce ESLint configuration and ESLint. If not, add these lines.

Reply:
So in this scenario, the .eslintrc.json may not be present either and may need to be added, same as in the following section.